crates/lib3h/src/dht/dht_protocol.rs

```diff
@@ -1,7 +1,8 @@
+use crate::dht::PeerAddress;
 use lib3h_protocol::{data_types::EntryData, Address};
 use url::Url;
 
-pub type FromPeerAddress = String;
+pub type FromPeerAddress = PeerAddress;
```

Comment:
Not sure how useful `FromPeerAddress` is at all. I think we should use a struct with a field of that name instead. Not related directly to this PR however.

Reply:
Yeah, I'm also on the fence for this one now